contracts/modules/staking/StakingProducts.sol

```diff
@@ -617,15 +616,15 @@ contract StakingProducts is IStakingProducts, MasterAwareV2, Multicall {
 
   function _setInitialProducts(uint poolId, ProductInitializationParams[] memory params) internal {
 
-    uint globalMinPriceRatio = cover().getGlobalMinPriceRatio();
+    uint defaultMinPriceRatio = cover().getDefaultMinPriceRatio();
     uint totalTargetWeight;
 
     for (uint i = 0; i < params.length; i++) {
 
       ProductInitializationParams memory param = params[i];
 
-      if (params[i].targetPrice < globalMinPriceRatio) {
-        revert TargetPriceBelowGlobalMinPriceRatio();
+      if (params[i].targetPrice < defaultMinPriceRatio) {
+        revert TargetPriceBelowDefaultMinPriceRatio();
```

Comment:
I'm assuming its safe to rename these errors from Global to Default? (i.e. none of our clients / system rely on these error names)

@shark0der correct?

Reply:
i don't see an issue with renaming. frontend will use the new abi and display the new error name if it happens (if they decode the errors anyway).